temporal-testing/README.md

```diff
@@ -44,4 +44,23 @@ and `io.temporal.testing.TestActivityExtension` for isolated testing of activiti
 
 See `io.temporal.testing.TestActivityEnvironment` that provides an easy way for isolated testing of
 activity implementations without needing to provide workflows calling the activities, triggering the workflows
-and bootstrapping a Temporal server or In-memory testing service.
\ No newline at end of file
+and bootstrapping a Temporal server or In-memory testing service.
+
+## To build a test service using GraalVM native-image
```

Comment:
By the way, I just found https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html if you wanted to wrap it up

Reply:
Thanks! We can prettify this later from a building point of view when we are sure that it will work for us at all. For now CLI is easier to make adjustments.